pychromecast/controllers/multizone.py

```diff
@@ -152,50 +210,36 @@ def register_listener(self, member_uuid, listener: MultiZoneManagerListener):
         listener.multizone_new_cast_status(group_uuid, cast_status)
             The group uuid, of which the cast is a member, has new status
         """
-        member_uuid = str(member_uuid)
-        if member_uuid not in self._casts:
-            self._casts[member_uuid] = {"listeners": [], "groups": set()}
-        self._casts[member_uuid]["listeners"].append(listener)
-
-    def deregister_listener(self, member_uuid, listener):
+        member_uuid_str = str(member_uuid)
+        if member_uuid_str not in self._casts:
+            self._casts[member_uuid_str] = {"listeners": [], "group_memberships": set()}
+        self._casts[member_uuid_str]["listeners"].append(listener)
+
+    def deregister_listener(
+        self, member_uuid: UUID, listener: MultiZoneManagerListener
+    ) -> None:
         """Deregister listener for audio group changes of cast uuid."""
         self._casts[str(member_uuid)]["listeners"].remove(listener)
 
-    def get_multizone_memberships(self, member_uuid):
+    def get_multizone_memberships(self, member_uuid: UUID) -> list[str]:
         """Return a list of audio groups in which cast member_uuid is a member"""
-        return list(self._casts[str(member_uuid)]["groups"])
+        return list(self._casts[str(member_uuid)]["group_memberships"])
 
-    def get_multizone_mediacontroller(self, group_uuid):
+    def get_multizone_mediacontroller(self, group_uuid: UUID) -> MediaController:
         """Get mediacontroller of a group"""
-        return self._groups[str(group_uuid)]["chromecast"].media_controller
-
-
-class MultiZoneControllerListener(abc.ABC):
-    """Listener for receiving audio group events."""
-
-    @abc.abstractmethod
-    def multizone_member_added(self, group_uuid: str):
-        """The cast has been added to group identified by group_uuid."""
-
-    @abc.abstractmethod
-    def multizone_member_removed(self, group_uuid: str):
-        """The cast has been removed from group identified by group_uuid."""
-
-    @abc.abstractmethod
-    def multizone_status_received(self):
-        """Multizone status has been updated."""
+        return self._groups[str(group_uuid)]["chromecast"].media_controller  # type: ignore[no-any-return]
```

Comment:
The mypy ignore will be removed in an upcoming PR which adds type annotations to the `Chromecast` class